script/mimic

```diff
@@ -0,0 +1 @@
+Subproject commit 203f85db5de57703e7100a00636060f3095dda29
```

Comment:
This isn't a proper submodule since there's no `.gitmodules` so it can't be fetched from the branch. This causes the mimic docker image to fail building (which you can see in the output on jenkins).